src/app/treasure-hunt/treasure-hunt-report/executive-summary/executive-summary.component.html

```diff
@@ -25,11 +25,16 @@
       [settings]="settings" [title]="'Detailed Summary'">
     </app-executive-summary-table>
   </div>
-  <div class="col-12">
+  <div class="col-12 col-lg-6">
     <app-carbon-emissions-summary-table
       [settings]="settings" [treasureHuntResults]="treasureHuntResults">
     </app-carbon-emissions-summary-table>
   </div>
+  <div class="col-12 col-lg-6">
```

Comment:
It would be nice to have unit labels on all these since there is a chart with different units now
![image (55)](https://user-images.githubusercontent.com/60401912/202545655-1ef98b46-b7ad-4d0d-b2cc-911d610f8622.png)